tests/e2e/test_e2e_template.py

```diff
@@ -604,7 +604,7 @@ def test_make_hypertrain(
     # Print wandb status for debugging reasons
     run("wandb status")
 
-    n = 2
+    n = 1
```

Comment:
`n = 2` flakes sometimes: while the second job is pending, the first completes the sweep and thus the second has nothing to run.